src/python/vdo-setup-dmtest/config.yaml

```diff
@@ -0,0 +1,28 @@
+# Devices to be used by dmtest-python
+DATA_DEVICE: /dev/mapper/scratch-vdo_scratch
+METADATA_DEVICE: /dev/mapper/scratch-vdo_scratch
+
+# Dmtest-python and dependency repos
+# May be defined with URL or full path. If URL, automatic cloning will be performed.
+DMTEST_PYTHON_REPO: https://github.com/dm-vdo/dmtest-python
+BUFIO_TEST_REPO: https://github.com/bmarzins/bufio-test.git
+BLK_ARCHIVE_REPO: https://github.com/jthornber/blk-archive.git
```

Comment:
why do we need blk_archive? What do we plan to use it for?

Reply:
Strictly speaking, we do not need bufio-test or blk-archive. I was asked to include them in the script so we could more easily explore if we want to use them in VDO tests we add to dmtest-python.

Reply:
This seems weird to me. Not sure why we would want to use them in our tests; especially blk-archive, which is an attempt to create a compress / dedupe type device as well. Seems like just excess time / space to d\l this stuff. But I'll not block if someone specifically asked for it.